src/bp-groups/bp-groups-template.php

```diff
@@ -7355,3 +7355,22 @@ function bp_get_group_current_invite_tab() {
 	 */
 	return apply_filters( 'bp_get_group_current_invite_tab', $tab );
 }
+
+/**
+ * Return string with the article prefix "a/an".
+ *
+ * @since BuddyBoss 1.2.8
+ *
+ * @param string $string  Text string for adding the article prefix.
+ * @return string
+ */
+function bp_get_article_prefix( $string ) {
+	if ( empty( $string ) ) {
+		return $string;
+	}
+
+	$vowelArry = array('a','e','i','o','u');  // array of vowel
+	$prefix = in_array( strtolower( substr( $string ,0,1 ) ), $vowelArry ) ? __( 'an', 'buddyboss' ) : __( 'a', 'buddyboss' );
+	return $prefix . ' ' . $string;
```

Comment:
```suggestion
	return apply_filters( 'bp_get_article_prefix ', $prefix . ' ' . $string );
```

Reply:
add a filter so if anyone has to make any changes then they can easily.

Reply:
Added filter as named `bp_get_article_prefix`.